src/gobapi/graphql_streaming/graphql2sql/graphql2sql.py

```diff
@@ -487,11 +491,21 @@ def _json_build_attrs(self, attributes: list, relation_name: str):
         :param relation_name:
         :return:
         """
-        return ",".join([f"'{to_snake(attr)}', {relation_name}.{to_snake(attr)}" for attr in attributes
-                         if attr not in self.srcvalues_attributes])
+        json_attrs = ",".join([f"'{to_snake(attr)}', {relation_name}.{to_snake(attr)}" for attr in attributes
+                              if to_snake(attr) not in self.srcvalues_attributes + self.relvalues_attributes])
+
+        if self._is_relvalue_requested(attributes):
+            rel_attrs = ",".join([f"'{to_snake(attr)}', rel_{self.relcnt}.{to_snake(attr).replace('_relatie', '')}"
+                                 for attr in attributes if to_snake(attr) in self.relvalues_attributes])
```

Comment:
to_snake(attr) is repeated several times, probably have a snake_attr variable instead?

Reply:
Added snake_attrs list